File: classes/parte-7-paradigmas/119-orientado-a-eventos-y-callbacks/implementaciones/python/main.py
import sys

recolectados = []


def al_evento(i):
    recolectados.append(i)


n = int(sys.stdin.readline())
for i in range(1, n + 1):
    al_evento(i)
print("eventos=" + "-".join(str(x) for x in recolectados))

File: classes/parte-7-paradigmas/119-orientado-a-eventos-y-callbacks/implementaciones/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	line, _ := bufio.NewReader(os.Stdin).ReadString('\n')
	n, _ := strconv.Atoi(strings.TrimSpace(line))
	var recolectados []string
	alEvento := func(i int) {
		recolectados = append(recolectados, strconv.Itoa(i))
	}
	for i := 1; i <= n; i++ {
		alEvento(i)
	}
	fmt.Printf("eventos=%s\n", strings.Join(recolectados, "-"))
}
